src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -434,6 +434,7 @@ def load_arguments(self, _):
         c.argument('rotation_poll_interval')
         c.argument('enable_sgxquotehelper', action='store_true')
         c.argument('enable_app_routing', action="store_true")
+        c.argument('enable_ai_toolchain_operator', is_preview=True, action='store_true')
```

Comment:
As the option is now added to official azure-cli, it should not go under preview anymore?

Reply:
Addressed.